### PR TITLE
Upgraded to TF 0.14 and issues fixes

### PR DESCRIPTION
**Pull request by SadriG91**

### CODE REVIEW

File: examples/default/init/main.tf [hunk 1]
```diff
@@ -13,8 +13,7 @@ terraform {
 }
 
 provider "aws" {
-  version             = ">= 2.17"
-  region              = "eu-west-1"
+  region = "eu-west-1"
```
Comment (larstobi) [inline examples/default/init/main.tf:16]: This is incorrectly formatted with whitespace. `terraform fmt`

Comment (SadriG91) [reply examples/default/init/main.tf]: this have been corrected

**Review by larstobi — CHANGES_REQUESTED** [review]
Looks good! Just a small whitespace thing. Should be discovered automatically by a check.

**Review by larstobi — APPROVED** [review]
LGTM

**Review by larstobi — APPROVED** [review]
LGTM

**Pull request merged**